src/target/adiv5_swd.c

```diff
@@ -153,6 +153,7 @@ bool adiv5_swd_scan(const uint32_t targetid)
 	}
 
 	dp->dp_low_write = firmware_dp_low_write;
+	dp->dp_low_read = firmware_dp_low_read;
 	dp->error = firmware_swdp_error;
```

Comment:
Given this change set, we propose fixing the naming of the error function here inline with the names of the ones being removed - `adiv5_swd_clear_error`.

Reply:
likewise good suggestion